pkg/image/api/helper.go

```diff
@@ -285,6 +286,16 @@ func SplitImageStreamTag(nameAndTag string) (name string, tag string, ok bool) {
 	return name, tag, len(parts) == 2
 }
 
+// StreamKeyFromTagRef returns the image stream key from an ImageStreamTag
+// object reference.
+func StreamKeyFromTagRef(ref kapi.ObjectReference) (string, bool) {
```

Comment:
I think this needs to be more strongly typed - Key to what?  I don't want random people using this without understanding what the key is for.